cmake/sca/dtdoctor/sca.cmake

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright The Zephyr Project Contributors
+
+include(python)
+include(kconfig)
+
+if(NOT "${ZEPHYR_TOOLCHAIN_VARIANT}" STREQUAL "zephyr")
+  message(WARNING "dtdoctor has only been tested with the Zephyr SDK toolchain.")
+endif()
+
+set(_DT_DIAG_WRAP ${ZEPHYR_BASE}/scripts/dts/dtdoctor_sca_wrapper.py)
```

Comment:
if this is intended to indicate a locally used variable, then please use lower case, and remove the `_`.

Although `_` can be used for vars, then a leading `_` in functions has special meaning in CMake, so better avoid leading `_` in general in CMake.